frontend/src/services/groups-api/groups-api.service.ts

```diff
@@ -1,5 +1,9 @@
 import { ApiPath, GroupsApiPath, HttpMethod } from 'common/enums/enums';
-import { EntityPagination, GroupsItemResponseDto } from 'common/types/types';
+import {
+  EntityPagination,
+  GroupsDeleteRequestParamDto,
+  GroupsItemResponseDto,
+} from 'common/types/types';
 import { Http } from 'services/http/http.service';
```

Comment:
Can be simplified with "services/services"

Reply:
http is not exported from services, is there any reason for that or that is just a mistake and I can add it to exports?

Reply:
services does not have the http export for some reason ...